semperfiwebdesign/all-in-one-seo-pack
# Disable review notice

Issue #2589

## Proposed changes

Since some users are still having problems with dismissible notices not staying dismissed, we should temporarily disable this one from happening.

## Types of changes

What types of changes does your code introduce?

- Removing old code/functionality

## Checklist


## Testing instructions
1. Just have plugin activated, and notice should not be active.

1. Activating plugin should not reset notice either.

Check SQL database. 

1. Go to wp_options > aioseop_notices.
2. Click edit.
3. Check to see if 'review_plugin' does not exist.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...
